docs/spfx/update-latest-packages.md

```diff
@@ -0,0 +1,116 @@
+
+In the `package.json` file, locate the package(s) under the `dependencies` and `devDependencies` section and update the version to the latest version available that was listed in the `npm outdated` command. For example, the image below highlights the version updates to SharePoint Framework packages, the left section referring to the old and the right section referring to the latest package versions.
+
+![Edit package versions in package.json file](../../images/npm-update-packagejson-versions.png)
+
+Once you have updated the package versions, Save the `package.json` file.
+
+#### Update packages
+Open your favorite console and navigate to the root of your project directory. Follow the instructions below to successfully update the packages to its latest version.
+
+Delete the `node_modules` folder. This is the folder where npm installs the packages locally for your project. Deleting this folder forces npm to fetch the latest and not duplicate existing packages.
+
+If you are using a Mac or a Linux environment, then run the following command:
+
+```
+rm -rf node_modules/
+```
+
+If you are using a Windows environment, then run the following command:
+
+```
+ rd /s /q node_modules/
+```
+
+Now, execute the following command to fetch the latest packages from the npm registry:
+
+```
+npm install
+```
+
+This command will create the `node_modules` folder and install all the packages your project depends on based on the information available in the `package.json` file. As we updated the file with the latest versions, npm will now have the latest packages installed. 
+
+Once the packages are installed, execute the following command to clean up any old build artifacts:
+
+```
+gulp clean
+```
+
+## Update yeoman generator for SharePoint
+Unlike the npm packages that are installed to a specific client-side project, the yeoman generator for SharePoint is installed globally in your machine.
+
+To find if the yeoman generator for SharePoint is out of date, run the following command in a console window. 
+
+```
+npm outdated -g
+```
+
+The command will list the following information about the packages installed globally in your machine. This information is looked up from the versions installed in your machine and the npm registry.
+
+* Current version installed globally in your machine
+* Version requested by you when you installed
+* Latest version available
+
+![NPM outdated global packages](../../images/npm-outdated-global-packages-list.png)
+
+To identify the generator package, look for the following package name:
+
+```
+@microsoft/generator-sharepoint
+```
+
+### Update package
+Open your favorite console and execute the following command to update the generator to its latest published version:
+
+```
+npm install @microsoft/generator-sharepoint@latest -g
+```
+
+The command will update the yeoman generator for SharePoint to the latest published version. You can validate this by executing the following command in the console:
```

Comment:
Should we just add here that it's updating the latest yeoman for SharePoint and all the needed dependencies - just add some additional clarity.